src/components/PostList/PostList.tsx

```diff
@@ -1,5 +1,16 @@
 import React from 'react';
 
-export const PostList: React.FC = () => (
-  <>Put the list here</>
+import { TotalPostInfo } from '../../types/TotalPostInfo';
+import { PostInfo } from '../PostInfo';
+
+type Props = {
+  posts: TotalPostInfo[];
+};
+
+export const PostList: React.FC<Props> = ({ posts }) => (
+  <ul className="PostList">
```

Comment:
remove default list styles - marks on the left side (for comments and posts):
![image](https://user-images.githubusercontent.com/86923089/214372632-64553384-b064-4512-a145-c2484a6d7c1a.png)